revolv/project/models.py

```diff
@@ -280,6 +281,19 @@ def mark_as_incomplete_project(self):
         self.save()
         return self
 
+    def update_categories(self, category_list):
+        """ Updates the categories list for the project.
+
+        :category_list The list of categories in the submitted form
+        """
+        # Clears all the existing categories
+        self.category_set.clear()
+
```

Comment:
Let me know your take on deleting the old connections and making new ones is. I figure that there are very few categories, so just readding all the categories is fine. I can change this to query and only insert / delete category objects if necessary.